server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -1,34 +1,34 @@
 # This compose file is useful for testing https.
 # The .env file sets ENVVARS for the Docker CLI used by this compose file.
 ---
-version: "3"
+version: "3.8"
 services:
   cms:
-    image: taccwma/frontera-cms:9877f12
+    image: taccwma/core-cms:latest
     volumes:
       - ../cms/secrets.py:/code/taccsite_cms/secrets.py
       - ../cms/uwsgi/uwsgi.ini:/code/uwsgi.ini
       - ../../cms/static:/code/static
       - ../../cms/media:/code/media
     command: python manage.py runserver 0.0.0.0:8000
-    container_name: frontera_prtl_cms
+    container_name: core_prtl_cms
```

Comment:
__Opinion.__ Any interest for `portal` instead of `prtl`? _We don't use `btn` instead of `button`, nor document "app" instead of "application". I dislike verbosity (even my own), but abbreviations are an unnecessary mental overhead._

Reply:
`portal` makes sense to me